src/plugins/about/index.js

```diff
@@ -0,0 +1,133 @@
+const log = require('Debug')('log:about');
+const trace = require('Debug')('trace:about');
+const debug = require('Debug')('debug:about');
+
+const bluebird = require('bluebird');
+const Listener = require('Listener');
+const execFile = require('child_process').execFile;
+//bluebird.promisify(execAsync);
+
+
+//private functions
+
+
+function execAsync(command,args){
```

Comment:
Thanks for the link, yea, that example uses the third party library 'child-process-promise', so I am assuming this is one that the bluebird solution requires a bit of hand rolling to handle.